be/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd be/ && npx lint-staged
```

Comment:
No hooks, please

Reply:
Sorry I thought no hooks for prettier, I removed them all